docs/3.0/custom-widgets.md

```diff
@@ -0,0 +1,67 @@
+---
+---
+# Custom Widgets
+
```

Comment:
I'm a bit confused on the wording here; there are custom Widgets and custom Components.  One is written entirely in JS and may be part of the codebase or loaded through npm; the other would be akin to a Cordova plugin (and probably leverage native APIs), right?  I would assume the JS ones are Widgets, since they would extend the `Widget` class or one of its descendants, though the native ones would do that as well.

Reply:
Yeah, the terminology kind of evolved like @cpetrov said. It could be argued that the native components are custom *widgets*  because they directly extend `Widget`, while JS-only *components* always have to have a class in-between which makes them a variation on *existing* widgets.

But yeah, the two names could be flipped, it's not a self-evident separation.